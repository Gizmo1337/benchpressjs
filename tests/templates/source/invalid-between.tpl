<script>
  var RELATIVE_PATH = "{relative_path}";
  var config = JSON.parse('{{configJSON}}');
  var app = {
    template: "{template.name}",
    user: JSON.parse('{{userJSON}}')
  };
</script>
