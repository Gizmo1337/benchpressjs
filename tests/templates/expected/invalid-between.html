<script>
  var RELATIVE_PATH = "";
  var config = JSON.parse('{}');
  var app = {
    template: "header",
    user: JSON.parse('{}')
  };
</script>
